aptos-move/framework/aptos-framework/sources/coin.move

```diff
@@ -401,6 +432,7 @@ module aptos_framework::coin {
         let name = string::utf8(b"Fake money");
         let symbol = string::utf8(b"FMD");
```

Comment:
All these tests are repeating the same setup lines. I think it's about time to move them into a setup method to reduce redundancy.